app/views/webhooks/index.html.erb

```diff
@@ -12,14 +12,24 @@
 
   <dl>
     <% Samson::Integration::SOURCES.each do |source| %>
-      <dt><%= source.titleize %>:</dt><dd><%= link_to_url(send("integrations_#{source}_deploy_url", @project.token)) %></dd>
+      <dt>
+        <%= source.titleize %>:</dt><dd><%= link_to_url(send("integrations_#{source}_deploy_url", @project.token)) %>
+        <% if source == 'github' && token = Integrations::GithubController.secret_token %>
+          <br/>
+          <% if current_user.super_admin? %>
```

Comment:
Think it really needs to be a super admin? Whoever's the admin of the project will likely be the one setting up the GitHub webhooks.  I think project admin for the current project is good enough.